### PR TITLE
Descriptive Link Names

### PR DESCRIPTION
**Pull request by JohanFreelancer9**
Issue #969

### CODE REVIEW

File: microsoft-365/compliance/service-encryption-with-customer-key-faq.md [hunk 1]
```diff
@@ -43,14 +43,14 @@ BYOK with Azure Information Protection for Exchange Online is offered in the Off
...
+No. Office 365 Message Encryption and the option to provide and control your own encryption keys with Bring Your Own Key (BYOK) for Azure Information Protection was not designed to respond to law enforcement subpoenas. Office 365 Message Encryption with BYOK for AIP was designed for compliance focused customers that need to meet their internal or external compliance obligations. Microsoft takes third-party requests for customer data very seriously. As a cloud service provider, we always advocate for the privacy of customer data. In the event we get a subpoena, we always attempt to redirect the third party to the customer to obtain the information. (Please read Brad Smith's blog: [Protecting customer data from government snooping](https://blogs.microsoft.com/blog/2013/12/04/protecting-customer-data-from-government-snooping/)). We periodically publish detailed information of the request we receive in the [Law Enforcement Requests Report](https://www.microsoft.com/en-us/corporate-responsibility/law-enforcement-requests-report).
```
Comment (mypil) [inline microsoft-365/compliance/service-encryption-with-customer-key-faq.md]:
```suggestion
No. Office 365 Message Encryption and the option to provide and control your own encryption keys with Bring Your Own Key (BYOK) for Azure Information Protection was not designed to respond to law enforcement subpoenas. Office 365 Message Encryption with BYOK for AIP was designed for compliance focused customers that need to meet their internal or external compliance obligations. Microsoft takes third-party requests for customer data very seriously. As a cloud service provider, we always advocate for the privacy of customer data. In the event we get a subpoena, we always attempt to redirect the third party to the customer to obtain the information. (Please read Brad Smith's blog: [Protecting customer data from government snooping](https://blogs.microsoft.com/blog/2013/12/04/protecting-customer-data-from-government-snooping/)). We periodically publish detailed information of the request we receive in the [Law Enforcement Requests Report](https://www.microsoft.com/corporate-responsibility/law-enforcement-requests-report).
```

File: microsoft-365/compliance/service-encryption-with-customer-key-faq.md [hunk 2]
```diff
@@ -43,14 +43,14 @@ BYOK with Azure Information Protection for Exchange Online is offered in the Off
...
+No. Customer Key was not designed to respond to law enforcement subpoenas. It was designed for regulated customers to meet their internal or external compliance obligations. Microsoft takes third-party requests for customer data very seriously. As a cloud service provider, we always advocate for the privacy of customer data. In the event we get a subpoena, we always attempt to redirect the third party to the customer to obtain the information. (Please read Brad Smith's blog: [Protecting customer data from government snooping](https://blogs.microsoft.com/blog/2013/12/04/protecting-customer-data-from-government-snooping/)). We periodically publish detailed information of the request we receive [Law Enforcement Requests Report](https://www.microsoft.com/en-us/corporate-responsibility/law-enforcement-requests-report).
```
Comment (mypil) [inline microsoft-365/compliance/service-encryption-with-customer-key-faq.md]:
```suggestion
No. Customer Key was not designed to respond to law enforcement subpoenas. It was designed for regulated customers to meet their internal or external compliance obligations. Microsoft takes third-party requests for customer data very seriously. As a cloud service provider, we always advocate for the privacy of customer data. In the event we get a subpoena, we always attempt to redirect the third party to the customer to obtain the information. (Please read Brad Smith's blog: [Protecting customer data from government snooping](https://blogs.microsoft.com/blog/2013/12/04/protecting-customer-data-from-government-snooping/)). We periodically publish detailed information of the request we receive [Law Enforcement Requests Report](https://www.microsoft.com/corporate-responsibility/law-enforcement-requests-report).
```

**Review by mypil — CHANGES_REQUESTED** [review]
@JohanFreelancer9 - Created this commit to remove the "en-us" from the links. Should this be the case for this PR? Please let me know your thoughts. Thank you.

**Comment by JohanFreelancer9** [comment]
@mypil  No, I initially also removed the locale, but then the link does not seem to work - certainly not in Vietnam.

**Comment by mypil** [comment]
@KCCross - PR has been copy edited and is ready for final review. This is based on issue #969.

Thank you.

cc: @kenwith

**Comment by mypil** [comment]
@KCCross - We would like to follow up. Any updates on this?

Thank you.

**Comment by KCCross** [comment]
File no longer exists. Content updated and en-us links removed.

**Pull request closed without merging**